packages/core/src/manifest.ts

```diff
@@ -48,7 +48,7 @@ function defaultManifest(): ManifestData {
   };
 }
 
-const MANIFEST_DEFAULT_DIR = '.openzeppelin';
+const MANIFEST_DEFAULT_DIR = process.env.MANIFEST_DEFAULT_DIR || '.openzeppelin';
```

Comment:
I think it would be better to use an environment variable name such as `OPENZEPPELIN_NETWORK_FILES_DIR` to more closely match with the ["Network Files"](https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files) term used in the documentation.  It may also be useful to make it load from a `.env` file.  For example:
```suggestion
require('dotenv').config();
const MANIFEST_DEFAULT_DIR = process.env.OPENZEPPELIN_NETWORK_FILES_DIR || '.openzeppelin';
```

Note this would also need a `"dotenv"` dependency in `packages/core/package.json`

Reply:
Hey, thanks for quick review @ericglau 
I did add `dotenv` to the dependencies but as there is no `.env` file I started to have second thoughts if adding library just for sake of one env variable is reasonable. I'm very reluctant to add new libraries to the codebase and always try to keep it vanilla.
Also, my rule of thumb is to refactor for third repeating element. We haven't reached that point. 

I'm open to hear, what you think about this. If you think this is good idea then no problem in implementing it but I don't think I should be making decision about it. ✌️

Reply:
I added section of documentation that explains this feature. Let me know if this is up to `openzeppelin` standards

Reply:
Thanks for the comments, I think the env var itself is fine as is.  I left a separate suggestion for the docs.